packages/rulesets/src/spectral/az-arm.ts

```diff
@@ -707,6 +707,44 @@ const ruleset: any = {
       },
     },
 
+    ///
+    /// ARM RPC rules for SystemData
+    ///
+
+    // RPC Code: RPC-SystemData-V1-01 and RPC-SystemData-V1-02
+    // Note that this rule is normally already covered when the common types Resource is used.
```

Comment:
What is already covering it? Another rule? Can we reference it in this comment?

Reply:
When the common-types resource definition is used there won't be any system data references because the common-types resource definition has system data. So, this rule is only applicable for specs that are not using the common-types resource definition. Updating the comment to clarify this.